### DISCUSSION TITLE
Authentication with Access Tokens using .NET Core backend

### DISCUSSION THREAD

**Discussion by LucaElbio** [adonisjs/core#4837] [Help]
Hi, i've developed an app using Adonis Framework authenticated with Access Tokens. I'd like to use Access Tokens to authenticate another app but this second app uses .NET Core as backend framework. My question is, could i and is it viable use Access Tokens as authentication provider? Thank you so much for your time!

**Comment by thetutlage** [comment]
Yes, its perfectly valid to use Access tokens for this use-case

**Reply by LucaElbio** [reply]
Great, my question now is how to do it, should i use my Adonis backend as the token manager or could i create and handle tokens directly from my .NET Core backend?
